mscore/preferences.cpp

```diff
@@ -1149,6 +1151,18 @@ void PreferenceDialog::resetShortcutClicked()
       active->setText(1, shortcut->keysToString());
       shortcutsChanged = true;
       }
+    
+void PreferenceDialog::saveShortcutListClicked()
+      {
+      QString saveFileName = QFileDialog::getSaveFileName(new QWidget(),"Save Shortcuts", tr("XML Files(*.xml)"));
```

Comment:
Do you have any suggestions about how this can be done differently? Should I not use getSaveFileName at all?